test/query_test.rb

```diff
@@ -11,4 +11,21 @@ def test_basic
     assert_equal ["Apple", "Milk"], query.execute.map(&:name).sort
   end
 
+  def test_indices_boost
+    store_names(["Rex"], Animal)
+    store_names(["Rexx"], Product)
+    index_names = [Animal.searchkick_index.name, Product.searchkick_index.name]
+    indices_boost = { Animal.searchkick_index.name => 1, Product.searchkick_index.name => 200 }
+
+    query = Store.search("Rex", index_name: index_names, indices_boost: indices_boost, execute: false)
```

Comment:
I think we can simplify the test significant. Rather than check the generated body, let's check the order of the results with `assert_order`.